### PR TITLE
[java] (doc) Fix typo: "an accessor" not "a"

### PR DESCRIPTION
**Pull request by igormoreno**
## Fix a small typo

Change from "a accessor" to "an accessor".

### CODE REVIEW

**Comment by pmd-test** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="true">This changeset introduces 0 new violations, 0 new errors and 0 new configuration errors,<br />
removes 0 violations, 2 errors and 2 configuration errors.<br />
<a href="https://chunk.io/pmd/39266e86b8d34d31aae3064cb44e306a/diff/index.html">Full report</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: pmd-java/src/main/resources/category/java/bestpractices.xml [hunk 1]
```diff
@@ -71,7 +71,7 @@ public class Outer {
           class="net.sourceforge.pmd.lang.java.rule.bestpractices.AccessorMethodGenerationRule"
           externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_bestpractices.html#accessormethodgeneration">
         <description>
-When accessing a private field / method from another class, the Java compiler will generate a accessor methods
+When accessing a private field / method from another class, the Java compiler will generate an accessor methods
```
Comment (adangel) [inline pmd-java/src/main/resources/category/java/bestpractices.xml:74]: That's plural: methods - should there be no article at all then?
-> ... the Java compiler will generate accessor methods

Or we make it singular
-> ... the Java compiler will generate an accessor method

**Review by adangel — COMMENTED** [review]
Thanks for the pull request.
Please see my comment - I tend to use no article at all here....

File: pmd-java/src/main/resources/category/java/bestpractices.xml (hunk 1 above)
Comment (igormoreno) [reply pmd-java/src/main/resources/category/java/bestpractices.xml:74]: Oh, right.

I think that if it's one accessor method for each field/method then maybe it should be:
"When accessing **a** private field / method from another class, the Java compiler will generate **an** accessor **method**"

If we're talking in general then I guess the plural is better:
"When accessing private **fields / methods** from another class, the Java compiler will generate accessor **methods**"

What do you think?

```suggestion
When accessing private fields / methods from another class, the Java compiler will generate accessor methods
```

Comment (adangel) [reply pmd-java/src/main/resources/category/java/bestpractices.xml:74]: I like the general case, thanks!

**Pull request merged**